molecule/builder/create.yml

```diff
@@ -8,38 +8,14 @@
     molecule_ephemeral_directory: "{{ lookup('env', 'MOLECULE_EPHEMERAL_DIRECTORY') }}"
     molecule_scenario_directory: "{{ lookup('env', 'MOLECULE_SCENARIO_DIRECTORY') }}"
     molecule_yml: "{{ lookup('file', molecule_file) | from_yaml }}"
+    image_hash: e9f1683b1fe756751e3ec6481860414988eceaa25663e2e98dd5223e6c3b2856
```

Comment:
Where does this hash come from? Can you make a note so it looks less like a magic number.